<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="ApplicationFrame" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SPS Hysteresis Prediction</string>
  </property>
  <property name="useRBAC" stdset="0">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="PlotWidget" name="widgetPlot" native="true"/>
    </item>
    <item>
     <widget class="PlotSettingsWidget" name="widgetSettings" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load_Model"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionPrediction_Analysis"/>
    <addaction name="action_Trim_View"/>
    <addaction name="separator"/>
    <addaction name="action_minimize"/>
    <addaction name="action_maximize"/>
    <addaction name="action_fullscreen"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionShow_Plot_Settings"/>
    <addaction name="actionContinuous_Data_Export"/>
    <addaction name="action_Clear_Buffer"/>
    <addaction name="separator"/>
    <addaction name="action_Clear_Reference"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_view"/>
   <addaction name="menuTools"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="action_minimize">
   <property name="text">
    <string>Minimize</string>
   </property>
  </action>
  <action name="action_maximize">
   <property name="text">
    <string>Maximize</string>
   </property>
  </action>
  <action name="action_fullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle Fullscreen</string>
   </property>
   <property name="toolTip">
    <string>Toggle Fullscreen</string>
   </property>
  </action>
  <action name="actionShow_Plot_Settings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Plot Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_Load_Model">
   <property name="text">
    <string>&amp;Load Model</string>
   </property>
  </action>
  <action name="actionPrediction_Analysis">
   <property name="text">
    <string>Prediction &amp;Analysis</string>
   </property>
  </action>
  <action name="action_Trim_View">
   <property name="text">
    <string>&amp;Trim View</string>
   </property>
  </action>
  <action name="actionContinuous_Data_Export">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Continuous Data Export</string>
   </property>
   <property name="toolTip">
    <string>Enable / Disable continuous data export. This will write cycle data to disk when new measured data arrives.</string>
   </property>
  </action>
  <action name="action_Clear_Buffer">
   <property name="text">
    <string>&amp;Clear Buffer</string>
   </property>
  </action>
  <action name="action_Clear_Reference">
   <property name="text">
    <string>&amp;Clear Reference</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ApplicationFrame</class>
   <extends>QMainWindow</extends>
   <header>accwidgets.app_frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotSettingsWidget</class>
   <extends>QWidget</extends>
   <header>sps_apps.hysteresis_prediction.widgets.plot_settings_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>sps_apps.hysteresis_prediction.widgets.plot_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_about</sender>
   <signal>triggered()</signal>
   <receiver>main_window</receiver>
   <slot>showAboutDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>main_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_minimize</sender>
   <signal>triggered()</signal>
   <receiver>main_window</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_maximize</sender>
   <signal>triggered()</signal>
   <receiver>main_window</receiver>
   <slot>showMaximized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_fullscreen</sender>
   <signal>triggered()</signal>
   <receiver>main_window</receiver>
   <slot>toggleFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
